examples/fabric-admin/device_manager/DeviceManager.cpp

```diff
@@ -33,6 +33,7 @@ namespace {
 // Constants
 constexpr uint32_t kSetupPinCode               = 20202021;
 constexpr uint16_t kRemoteBridgePort           = 5540;
+constexpr uint16_t kLocalBridgePort            = 5540;
```

Comment:
Won't this need to be configurable based on argument if we plan to have two bridges running on the same host?

Reply:
Track in https://github.com/project-chip/connectedhomeip/issues/35030